src/infrastructure/persistence/PsqlUserRepository.ts

```diff
@@ -27,13 +32,13 @@ export class PsqlUserRepository implements UserRepository {
   async save(user: User): Promise<User> {
     await this.db.raw(
       `insert into "${USER_TABLE_NAME}" (id, email, creation_time, modification_time)
-        values (:id, :email, :creation_time, :modification_time)
+        values (:id, :email, :creationTime, :modificationTime)
         on conflict do nothing`,
       {
         id: user.id.value,
         email: user.email.value,
-        creation_time: user.creationTime,
-        modification_time: user.modificationTime,
+        creationTime: user.creationTime,
+        modificationTime: user.modificationTime,
```

Comment:
Don't think you want to change this one?

Reply:
`modification_time: user.modificationTime,`

Reply:
I guess it doesn't matter as you also changed in the statement

Reply:
It was actually intentional.. Just not sure where to draw the line 😇

Reply:
In other repositories (and actually later on in this one), we use underscores on the left, and camelCase on the right.  But it's not a big deal.

Reply:
Actually, not later on this one, it's not a raw statement.